src/TestRunnerContext.php

```diff
@@ -60,41 +60,51 @@
 class TestRunnerContext implements SnippetAcceptingContext
 {
     /** @var Filesystem $filesystem */
-    private $filesystem;
+    protected $filesystem;
 
     /** @var string $workingDirectory Place for generated tests */
-    private $workingDirectory;
+    protected $workingDirectory;
 
     /** @var string $documentRoot Root directory of the web server */
-    private $documentRoot;
+    protected $documentRoot;
 
     /** @var Process[] $processes Active processes */
-    private $processes = [];
+    protected $processes = [];
 
     /** @var Process $behatProcess Active behat process */
-    private $behatProcess;
+    protected $behatProcess;
 
     /** @var string $browserCommand */
-    private $browserCommand;
+    protected $browserCommand;
 
     /** @var ProcessFactory $processFactory */
-    private $processFactory;
+    protected $processFactory;
+
+    /** @var string[] List of created files */
+    protected $files = [];
+
+    /** @var bool */
+    protected $isAutogeneratedWorkingDirectory;
 
     /**
      * TestRunnerContext constructor.
      *
      * @param string|null         $browserCommand Shell command which executes the tester browser
+     * @param string|null         $workingDirectory
      * @param Filesystem|null     $fileSystem
      * @param ProcessFactory|null $processFactory
      */
     public function __construct(
         $browserCommand = null,
+        $workingDirectory = null,
```

Comment:
should I move this to be the last parameter? would look weird, but that way if someone already extended this class then it would not fail on `parent::__construct`. what do you think?

Reply:
I would prefer to add it as the last constructor argument.

Reply:
moved